sample_factory/envs/mujoco/mujoco_params.py

```diff
@@ -1,13 +1,80 @@
 def mujoco_override_defaults(env, parser):
     parser.set_defaults(
+        batched_sampling=True,
+        num_workers=8,
+        num_envs_per_worker=16,
+        worker_num_splits=2,
+        train_for_env_steps=1000000,
+
         encoder_type='mlp',
         encoder_subtype='mlp_mujoco',
         hidden_size=64,
         encoder_extra_fc_layers=0,
         env_frameskip=1,
-        nonlinearity='tanh'
+        nonlinearity='tanh',
+        batch_size=1024,
+        # num_batches_per_epoch=10,
+
+        use_rnn=False,
+        adaptive_stddev=False,
+        policy_initialization='torch_default',
+        reward_scale=0.1,
+        rollout=16,
+        max_grad_norm=0.0,
+        ppo_epochs=10,
+        ppo_clip_ratio=0.2,
+        value_loss_coeff=2.0,
+        exploration_loss_coeff=0.0,
+        learning_rate=3e-4,
+        lr_schedule='kl_adaptive_minibatch',
+        lr_schedule_kl_threshold=0.08,
```

Comment:
I'm fine with using adaptive LR, but stable baselines probably doesn't use this? Or do they?

Reply:
They used a linearly decaying LR but I didn't see an option for that in sample factory. Is there a way to do that or should I switch to a constant LR?

Reply:
RL-games uses adaptive learning rate though